backend/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: '2.2'
+
+services:
+  versewise-cms-backend:
+    container_name: versewise-cms-backend
+    build:
+      context: .
+    ports:
+      - "${HOST_PORT}:5000"
```

Comment:
HOST_PORT isn't a good name here, I guess my comment kinda confused you. I didn't mean to name it HOST_PORT I was using the name to give you the idea that the port is on the host, so rename this to a better name, like CMS_API_PORT. 

- [ ] Rename environment variable